### PR TITLE
Metricfrequencyprocessor Readme

### PR DESCRIPTION
**Pull request by ppawelecsumo**

### CODE REVIEW

File: pkg/processor/metricfrequencyprocessor/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## <a name="metricfrequencyprocessor"></a>Metric Frequency Processor
```
Comment (pmalek-sumo) [inline pkg/processor/metricfrequencyprocessor/README.md:1]:
```suggestion
## Metric Frequency Processor
```

I don't believe that's needed unless I'm missing something.

File: pkg/processor/metricfrequencyprocessor/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+### Config
+
+- `min_point_accumulation_time` - warm up time for processor. Processor won't sift any data point from a metric with no
+  earlier data point older than this value.
+- `constant_metrics_report_frequency` - minimum time between reports of a constant metric.
+- `low_info_metrics_report_frequency` - minimum time between reports of a low info metric.
+- `max_report_frequency` - minimum time between reports of any metric.
+
+*Low info definition*
```
Comment (pmalek-sumo) [inline pkg/processor/metricfrequencyprocessor/README.md:24]:
```suggestion
#### Low info definition
```

File: pkg/processor/metricfrequencyprocessor/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+- `iqr_anomaly_coefficient` - relative deviation from interquartile range which constitutes an anomaly.
+- `variation_iqr_threshold_coefficient` - variation to iqr quotient under which a metric is considered low info.
+
+*Data point caching*
```
Comment (pmalek-sumo) [inline pkg/processor/metricfrequencyprocessor/README.md:29]:
```suggestion
#### Data point caching
```

File: pkg/processor/metricfrequencyprocessor/README.md (hunk 1 above)
Comment (ppawelecsumo) [reply pkg/processor/metricfrequencyprocessor/README.md:1]: This pattern is present in other our processors, so I kept it. Any reason not to have it here?

Comment (ppawelecsumo) [reply pkg/processor/metricfrequencyprocessor/README.md:1]: this doc can be linked to with it, going (scrolling) directly to documentation and not github page with files

Comment (ppawelecsumo) [reply pkg/processor/metricfrequencyprocessor/README.md:1]: tbf, I see we achieve the same with a link straight to README file instead

so perhaps the authors had something different in mind

Comment (pmalek-sumo) [reply pkg/processor/metricfrequencyprocessor/README.md:1]: You don't need the `a` tag for that: 

> README.md#metric-frequency-processor

Comment (ppawelecsumo) [reply pkg/processor/metricfrequencyprocessor/README.md:1]: ok, then I don't see reason for this as well, removing

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**